specifyweb/frontend/js_src/lib/components/FormPlugins/LatLongUi.tsx

```diff
@@ -105,8 +105,7 @@ function Coordinate({
      * string to float
      */
     resource.set(coordinateField, parsed?.asFloat() ?? null, { silent: true });
-
-    resource.set(coordinateTextField, trimmedValue);
+    resource.set(coordinateTextField, trimmedValue, { silent: true });
```

Comment:
now every resource.set in here has silent, which means unload protect won't be triggered even if the user did make a change?

to fix this, consider removing `{ silent: true }` from the `resource.set(coordinateField` - that one shouldn't be affected by this bug as the decimal value should remain the same before and after being reformatted (hopefully)

Reply:
Fixed in https://github.com/specify/specify7/pull/3959/commits/d0788a9fe96135040234e773fe40a7f186b7c0dd